lib/components/narrative/default/default-itinerary.js

```diff
@@ -109,22 +112,20 @@ const ITINERARY_ATTRIBUTES = [
     render: (itinerary, options) => {
       const formattedStartTime = itinerary.startTime + options.offset
       const formattedEndTime = itinerary.endTime + options.offset
-      if (options.isSelected) {
-        if (options.selection === 'ARRIVALTIME') {
-          return <FormattedTime value={formattedStartTime} />
-        } else {
-          return <FormattedTime value={formattedEndTime} />
-        }
+
+      if (options.selection === 'ARRIVALTIME') {
+        return <FormattedTime value={formattedEndTime} />
       }
-      return (
-        <FormattedMessage
-          id='common.time.departureArrivalTimes'
```

Comment:
Do we still use this message id elsewhere? If not remove it from language files.

Reply:
Yes it's still used in the tabbed itineraries and I think it makes sense there!